v63003/u_inref/gtm8779.csh

```diff
@@ -15,49 +15,80 @@
 #
 
 $gtm_tst/com/dbcreate.csh mumps 3 >>& create.out
+echo "# Initially, all regions set to OFF"
```

Comment:
The echo should happen AFTER the "if ($status)" check. Since $status from the dbcreate call gets reset after the echo call.